credhub-uaa/credhub-uaa.yml

```diff
@@ -0,0 +1,178 @@
+            secret: "((uaa_admin_client_secret))"
+          credhub_cli:
+            authorized-grant-types: password,refresh_token
+            authorities: uaa.none
+            scope: credhub.read,credhub.write
+            secret: "" # credhub expects this to be empty
+            access-token-validity: 120
+            refresh-token-validity: 1800
+            override: true
+          concourse_client:
+            authorized-grant-types: client_credentials
+            authorities: credhub.read,credhub.write
+            scope: credhub.read,credhub.write
+            secret: "((concourse_credhub_client_secret))"
+            access-token-validity: 120
+            refresh-token-validity: 1800
+            override: true
+        jwt:
+          policy:
+            active_key_id: key-1
+            keys:
+              key-1:
+                signingKey: "((uaa_jwt_signing_key.private_key))"
+        scim:
+          users:
+            - name: admin
+              password: "((cf_admin_password))"
+              groups:
+                - uaa.admin
+            - name: credhub
+              password: "((credhub_user_password))"
+              groups:
+                - credhub.read
+                - credhub.write
+        sslCertificate: "((uaa_ssl.public_key))"
+        sslPrivateKey: "((uaa_ssl.private_key))"
+        url: "((credhub_url))"
+      uaadb:
+        address: 104.154.192.148
+        tls: disabled
+        databases:
+        - name: uaadb
+          tag: uaa
+        db_scheme: postgres
+        port: 5432
+        roles:
+        - name: uaaadmin
+          password: "((uaaadmin_db_password))"
+          tag: admin
+  - name: bpm
+    release: bpm
+  - name: credhub
+    release: credhub
+    properties:
+      credhub:
+        port: 8844
+        tls:
+          certificate: "((credhub_ssl.public_key))"
+          private_key: "((credhub_ssl.private_key))"
+        data_storage:
+          type: postgres
+          username: credhubadmin
+          password: ((credhubadmin_db_password))
+          host: 104.154.192.148
+          port: 5432
+          database: credhub
+          require_tls: false
+        authentication:
+          uaa:
+            ca_certs:
+              - "((uaa_ssl.public_key))"
+            enabled: true
+            url: "((credhub_url))"
+        encryption:
+          keys:
+            - provider_name: internal-provider
+              key_properties:
+                encryption_password: "((credhub_encryption_key))"
+              active: true
+          providers:
+            - name: internal-provider
+              type: internal
+        authorization:
+          acls:
+            enabled: true
+          permissions:
+            - path: /*
+              actors:
+                - uaa-client:concourse_client
+                - uaa-client:credhub_cli
+                - uaa-user:7ad5b8f7-fe2f-44a0-bc08-b8ab03435c68
```

Comment:
GUID here makes me nervous - this is some snowflake configuration inside the UAA database? Might at least be better as a variable... but I'm a bit out of my depth here.

Reply:
We could remove this and just include this as a setup step. It's not needed to get uaa up. Pretty sure we can use the credhub user just fine. The account should be able to grant itself access to the path with the `write_acl` permission.

```suggestion
```